src/components/routers/WebhookRouter.tsx

```diff
@@ -123,7 +123,7 @@ export default class WebhookForm extends React.Component<WebhookRouterFormProps,
             return map;
         }, {});
 
-        const uuid: string = (this.props.action && this.props.action.uuid) || generateUUID();
+        const { uuid } = this.props.action;
```

Comment:
No need to check for an existing action or a `UUID` on that action since `NodeEditor`'s `getAction` method ensures we're provided with a `UUID`.